htdocs/compta/bank/treso.php

```diff
@@ -184,7 +184,7 @@
 				$tmpobj->objid = $sqlobj->objid;
 				$tmpobj->ref = $sqlobj->ref;
 				$tmpobj->total_ttc = $sqlobj->total_ttc;
-				$tmpobj->type = $sqlobj->type;
+				$tmpobj->type = !empty($sqlobj->type);
```

Comment:
$tmpobj->type must be a string.  May be
`$tmpobj->type = empty($sqlobj->type) ? '' : $sqlobj->type;`